iphone/Classes/TiUISearchBarProxy.m

```diff
@@ -89,6 +89,15 @@ - (void)setSearchBar:(UISearchBar *)searchBar
 
   // In UISearchController searchbar is readonly. We have to replace that search bar with existing search bar of proxy.
   [(TiUISearchBar *)[self view] setSearchBar:searchBar];
+
+  // Set search bar properties to new search bar
+  NSDictionary *properties = [self allProperties];
+  for (NSString *key in properties.allKeys) {
```

Comment:
I assume `[self allProperties]` (`dynprops`) cannot be `nil`, otherwise the app could crash here.